### DISCUSSION TITLE
How to input CL efflux,vitro of transporter in PKSim

### DISCUSSION THREAD

**Discussion by dat994** [Open-Systems-Pharmacology/Forum#1837] [Models]
Hello every one
I am developing a PBPK model for a drug. The drug has scarce data for the transporter. Only 1 study published kinetic data for this drug but they publish it in CL efflux,vitro. I couldn't find a way to input this parameter into the PKsim. The software only allow for the input of Km and Vmax
Do you happen to know how? If PKsim is not possible. Can you suggest free PBPK software to implement this index in the PBPK model? 
Thank you everyone in advance

**Comment by StephanSchaller** [comment]
Hi @dat994, it dipends. Can you let us know what is the reported value and unit for `CL efflux,vitro`?

**Reply by dat994** [reply]
Hello Stephan, the 2 values are 16 and 2.1 and the unit is (μl/min/mg)

**Reply by StephanSchaller** [reply]
Then you also need incubation compound concentration.
You can then use, e.g.:
![image](https://github.com/user-attachments/assets/01f850c0-f2e2-43a3-90ff-6691d2d5f3c7)

and divide your value by incubation concentration, then double the value and use it as Vmax and use incubation concentration as Km.

Do you know the amount of transfected transporter?

Best,
Stephan

**Reply by dat994** [reply]
Thank you so much, this study investigates transporter of linezolid and it incubates the concentration of 10 μM.

**Reply by dat994** [reply]
Hello Stephan, Sorry I forgot to reply to all of the discussion content. Follow your suggestion if CL intrinsic is 16 and LZD conc is 10 then the Km is 10 μM and Vmax is (16/10)*2 = 3.2 nmol/min/pmol. is this correct? and I am not quite sure about the amount of transfected transporter you asked. The study used transfected cell line so what do you mean the amount of transfected transporter? thank you

**Reply by dat994** [reply]
And can you share with me the source or document for this calculation? it is needed to convinced the reviewer
